### DISCUSSION TITLE
Drifters not moving with lower resolution data

### DISCUSSION THREAD

**Discussion by Tstolarz** [OpenDrift/opendrift#1056] [Q&A] [answered]
Hello. I am working with high frequency radar surface current data using the OceanDrift model using two datasets. The medium resolution 13MHz 2km data and the lower resolution higher range 5MHz 6km data. Using the 2km data I am able to run the model and get the drifters to move, however with the 6km data the drifters do not move. This is despite the animation showing that the background x and y surface current velocities change with the drifters inside the data boxes.

Any ideas on why the drifters aren't moving? I've included the notebooks for my runs.

2km: https://notebooksharing.space/view/68632b11ee121b1c3caf2c41a854e75e21140c4b8ab6308a2a77b9fdccb14f9e#displayOptions=
6km: https://notebooksharing.space/view/12f024960021f008085083a2ec72c6191579a16804b5ba7ce319631383bd6421#displayOptions=

**Comment by Tstolarz** [comment, marked as answer]
Nevermind. I realized the units metadata was actually in m/s on the 6km data unlike the 2km data which was in cm/s and required /100 to get it to work correctly. Feel free to delete this thread or keep it as a reminder to always ALWAYS check the units of your data.

**Reply by jamespatrickmanning** [reply]
I am having similar issue with getting particles to move in DOPPIO fields. Do you have any example code that moves particles through DOPPIO? I noticed there is no "x(&Y)_seawater_velocity" when I use the native ROMS reader.

**Reply by jamespatrickmanning** [reply]
Finally, I found the issue was is the url I was reading.  I needed to read the individual DOPPIO nc files rather than the compiled history best.
